src/Application/Actions/Campaigns/GetSummariesForCharity.php

```diff
@@ -39,7 +39,7 @@ public function __construct(
             $args['charitySalesforceId'] ?? throw new HttpNotFoundException($request)
         );
 
-        $charity = $this->charityRepository->findOneBySfIDOrThrow($sfId);
+        $charity = $this->charityRepository->findOneBySalesforceIdOrThrow($sfId);
```

Comment:
function renamed for consistency with `CampaignRepository::findOneBySalesforceId`